probe_scraper/ping_expiry_alert.py

```diff
@@ -0,0 +1,289 @@
+import argparse
+import re
+from collections import defaultdict
+from datetime import date, datetime
+from typing import Any, Dict, List, Tuple
+
+import requests
+from google.cloud import bigquery
+
+from probe_scraper.emailer import send_ses
+
+EMAIL_SUBJECT_TEMPLATE = "Glean Pings Expiring for {app_name}"
+
+EMAIL_TEMPLATE = """
+The following BigQuery tables are set to start deleting collected data soon based on the retention policies.
+
+{tables}
+
+What to do about this:
+
+1. If this is expected and you do not need data for any of these tables past the listed dates, then no action is needed.
+
+2. If you wish to continue collecting data for a longer period of time for any of these tables, please file Jira ticket for the data org, requesting the changes to retention settings at https://mozilla-hub.atlassian.net/secure/CreateIssue.jspa?issuetype=10180&pid=10057
+
+Retention policies are defined in probe-scraper [1], with options to either stop collecting data after a certain or delete data older than the specified number of days.
+
+For more information on requesting help from the data team, see https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/6849444/Getting+Help+Jira+documentation
+If you have any questions, please ask on the #data-help Slack channel. We'll give you a hand.
+
+Your Friendly, Neighborhood Data Team
```

Comment:
We seem to be inconsistent in usage of `,` in our templates. I'd recommend not having one:
```suggestion
Your Friendly Neighborhood Data Team
```